index.js

```diff
@@ -11,6 +11,7 @@ nconf
   .file({ file: `${__dirname  }/config.json` });
 
 const app = express();
+
 const db = monk(nconf.get('MONGODB_URL'));
 db.options.multi = true;
 
```

Comment:
the line below this one, where BASE_URL is defined, it should load the value from config.json